internal/response-consumer/handler.go

```diff
@@ -81,7 +83,7 @@ func (this *handler) onMessage(ctx context.Context, msg *k.Message) {
 			Where("account = ?", value.Account).
 			Where("correlation_id = ?", correlationId)
 
-		if selectResult := baseQuery.Select("id").First(&run); selectResult.Error != nil {
+		if selectResult := baseQuery.Select("id", "response_full").First(&run); selectResult.Error != nil {
```

Comment:
Also, in case of partial response we cannot infer the run status just by looking at the partial upload like we do with `status = inferSatStatus(value.SatEvents, nil)`

Imagine we get 3 uploads for a given run where:
upload 1 - host 1 failed
upload 2 - host 2 failed
upload 3 - host 3 succeeded + playbook_run_completed

by doing an isolated `status = inferSatStatus(value.SatEvents, nil)` on upload 3 the playbook run will appear as successful even though it clearly is not.

Reply:
I added some logic to handle the edge cases for partial responses. It seems to be working as expected. I am not sure if I missed anything, tbh.